utils/custom_status.ts

```diff
@@ -3,18 +3,14 @@
 import {getMyPreferences} from 'mattermost-redux/selectors/entities/preferences';
 
 import {getPreferenceKey} from 'mattermost-redux/utils/preference_utils';
+import {Preferences} from 'mattermost-redux/constants';
 
 import {GlobalState} from 'types/store';
-import {Preferences} from 'utils/constants';
 
 function showCustomStatusPulsatingDotAndPostHeader(state: GlobalState) {
```

Comment:
Should we move these utils to selectors?

Reply:
We can but I think selectors are only used to get something from state and return it... If we move these to selectors then meaning of selectors will be changed.

Reply:
Update this as discussed.